### PR TITLE
Example: DC PseudoSection Simulation

### PR DESCRIPTION
**Pull request by fourndo**
Add DC_Pseudo_Section example.

### CODE REVIEW

**Comment by lheagy** [comment]
for this example to run natively in SimPEG, we need to merge in simpegDCIP into simpeg. @rowanc1 @sgkang, can we do this soon!?

File: SimPEG/Examples/DC_PseudoSection_Simulation.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    hzind = [(dx,15,-1.3),(dx, 15)]
+    
+    mesh = Mesh.TensorMesh([hxind, hyind, hzind], 'CCN')
+    
+    
+    # Set background conductivity
+    model = np.ones(mesh.nC) * sig[0]
+    
+    # First anomaly
+    ind = Utils.ModelBuilder.getIndicesSphere(loc[:,0],radi[0],mesh.gridCC)
```
Comment (lheagy) [inline SimPEG/Examples/DC_PseudoSection_Simulation.py]: It doesn't look like this Util is on this branch, could you add it??

**Comment by fourndo** [comment]
Fixed the broken link to Utils and tested locally. Should be good to go.

**Comment by lheagy** [comment]
Thanks! I will add the auto-docs for it then we can merge it in. We can move the getIndicesSphere into the base utils if you think it will be a handy function

**Comment by rowanc1** [comment]
@lheagy I am looking through DCIP branch. I am not quite sure whether @sgkang's updates have made it into the most recent PR. I will investigate a bit more and then pull it into a branch over here.

**Comment by lheagy** [comment]
kk, it looks to me like the 'IP' branch is essentially master in simpegDC, is that right @sgkang ?

**Comment by rowanc1** [comment]
@fourndo @lheagy this is included in the DCIP/dev PR. #236

**Pull request closed without merging**